lib/net/http/response.rb

```diff
@@ -273,7 +273,7 @@ def code_type   #:nodoc:
 
   def error!   #:nodoc:
     message = @code
-    message += ' ' + @message.dump if @message
+    message += " #{@message.dump}" if @message
```

Comment:
This will remove another string allocation, right?

```suggestion
    message = "#{message} #{@message.dump}" if @message
```